src/pdc-api.ts

```diff
@@ -132,23 +135,9 @@ const uploadUsingPresignedPost = async (
 	}).then(throwIfResponseIsNotOk);
 };
 
-interface ApiBulkUploadsPostRequest {
-	fileName: string;
-	sourceKey: string;
-}
-
-interface ApiBulkUploadsPostResponse {
-	id: number;
-	fileName: string;
-	fileSize: number;
-	sourceKey: string;
-	status: string;
-	createdAt: string;
-}
-
 const useRegisterBulkUploadCallback = () => {
-	const api = usePdcCallbackApi<ApiBulkUploadsPostResponse>('/bulkUploads');
-	return (params: ApiBulkUploadsPostRequest) =>
+	const api = usePdcCallbackApi<BulkUpload>('/bulkUploads');
+	return (params: WritableBulkUpload) =>
```

Comment:
While we do want this change, do we have to make it as part of this same commit? I.e., is the proposal work broken if we don't also convert the writable bulk uploads here? If not, please move this to a separate commit.

Reply:
(It arguably should have been part of my `BulkUpload` type conversion — and if it can be, great, but it's also fine as a separate commit.)